src/templates/finops-hub/createUiDefinition.json

```
@@ -561,6 +582,11 @@
     "outputs": {
       "hubName": "[basics('hubName')]",
       "location": "[location()]",
+      "storageSku": "[basics('storageSku')]",
+      "enableInfrastructureEncryption": "[steps('advanced').enableInfrastructureEncryption]",
+      "tagsByResource": "[steps('tags').tagsByResource]",
+      "skipEventGridRegistration": "[steps('eventGrid').skipEventGridRegistration]",
+      "eventGridLocation": "[steps('eventGrid').eventGridLocation]"
```

Most of these are already covered or were removed in other changes.
```suggestion
      "enableInfrastructureEncryption": "[steps('advanced').enableInfrastructureEncryption]",
```